src/App.jsx

```diff
@@ -1,9 +1,99 @@
 import './App.css'
+import ButtonCard from "./assets/components/ButtonCard.jsx";
+import ProductCard from "./assets/components/ProductCard.jsx";
+import Tile from "./assets/components/Tile.jsx";
+
+
+const toTheCollection = () => {
+    console.log('Naar de collectie geklikt');
+};
+
+const shopAllBags = () => {
+    console.log('shop all bags geklikt');
+};
+
+const preOrders = () => {
+    console.log('pre orders geklikt');
+};
+
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  )
+    return (<>
+
+
+        <h1>Handbags & Purses</h1>
+
+        <nav>
+            <ButtonCard onClick={toTheCollection}
+                        buttonText="to the collection"/>
+
+            <ButtonCard onClick={shopAllBags}
+                        buttonText="shop all bags"/>
+
+            <ButtonCard onClick={preOrders}
+                        disabled={true}
+                        buttonText="pre-orders"/>
+        </nav>
+
+
+        <main>
+            <ProductCard
+                BestSeller={"Best seller"}
```

Comment:
volgens mis is het tegen de schrijwijze als de properties onder een component beginnen met een hoofdletter.